### ISSUE TITLE
AliasOrder: Nested aliases start a new group

### ISSUE THREAD

**Issue by xxdavid** [rrrene/credo#1218]
Hi, thank you for making and maintaining Credo for so many years!

I think I found a little bug/discrepancy in the `Credo.Check.Readability.AliasOrder` check. Since 533b079 (#582), nested aliases (multi aliases) always start a new group of aliases.

For example in this module
```elixir
defmodule MyApp.MyModule do
  alias Code.Fragment
  alias IO.ANSI
  alias MyApp.Core.{User, Account}
  alias MyApp.Web.{ErrorHTML, PageController}
  alias Phoenix.Controller
end
```

the groups are
*  `Code.Fragment`, `IO.ANSI`
* `MyApp.Core.{User, Account}`
* `MyApp.Web.{ErrorHTML, PageController}`, `Phoenix.Controller`

instead of the expected single group with all the aliases. 

This results in an inconsistency where
```elixir
defmodule MyApp.MyModule do
  alias MyApp.Core.{Account, User}
  alias Code.Fragment
end
```
is reported but
```elixir
defmodule MyApp.MyModule do
  alias Phoenix.Controller
  alias MyApp.Core.{Account, User}
end
```
is not.

My guess is that commit 533b079 introduced the `meta[:line] + 1` because of multi-line nested aliases like this
```elixir
defmodule MyApp.MyModule do
  alias Code.Fragment
  alias IO.ANSI
  alias MyApp.Core.{
    User,
    Account
  }
  alias Phoenix.Controller
end
```
so that the nested alias breaks the group and there are three groups:
* `Code.Fragment`, `IO.ANSI`
* `MyApp.Core{User, Account}`
* `Phoenix.Controller`

But this does not seem very relevant to me anymore as since Elixir 1.6 the formatter automatically formats the code to
```elixir
defmodule MyApp.MyModule do
  alias Code.Fragment
  alias IO.ANSI

  alias MyApp.Core.{
    User,
    Account
  }

  alias Phoenix.Controller
end
```
where the groups are explicit.

I suggest either using the old `meta[:line]` without the `+1` (i.e., reverting 533b079) or using the line of the first nested module (`User` in my example) which would also act like  `meta[:line] + 1` in case of multi-line aliases but like plain `meta[:line]` in single-line nested aliases. What do you think? I can send a PR if wanted.

**Comment by rrrene** [comment]
Thx for reporting! :+1:

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It should be fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by xxdavid** [comment]
Looks good to me! Thank you for fixing it! 🙂
